R/recipe_run.R

```diff
@@ -507,6 +507,14 @@ orderly_prepare_data <- function(config, info, parameters, envir) {
   } else {
     hash_resources <- NULL
   }
+  missing_packages <- setdiff(info$packages, .packages(TRUE))
+  
+  if (length(missing_packages) > 0) {
+    orderly_log("packages",
+                sprintf("%s", missing_packages)) 
+    stop(paste("Missing packages:", 
+               paste(missing_packages, collapse = ", ")))
```

Comment:
I think that most errors `squote` the arguments (so here this would be `paste(squote(missing_packages, collapse = ", "))` for slightly nicer printing (this will affect the strings in your tests)